File: src/main/java/com/ambition/business/user/vo/SysUserVo.java
package com.ambition.business.user.vo;

import com.ambition.business.user.domain.SysUser;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.experimental.Accessors;

/**
 * <pre>
 *
 *
 * @title: SysUserVo
 * @description:
 * @company:
 * @author: wuys
 * @datetime: 2019-08-06 16:03
 * </pre>
 */
@Data
@EqualsAndHashCode(callSuper = false)
@Accessors(chain = true)
public class SysUserVo extends SysUser {

	private Long userId;
}

File: myBlogServer/src/main/java/com/ambition/business/user/vo/SysUserVo.kt
package com.ambition.business.user.vo

import com.ambition.business.user.domain.SysUser

/**
 * <pre>
 *
 *
 * @title: SysUserVo
 * @description:
 * @company: 润投科技
 * @author: wuys
 * @datetime: 2019-12-19 10:35
 * </pre>
 */
open class SysUserVo : SysUser() {
    var userId: Long? = null
}